safe-global/web-core
# fix: token amount alignment

## What it solves

Resolves #702

## How this PR fixes it

`vertical-alignment` is only applied when a token `logoUri` is present in the `TokenAmount` component

## How to test it

View a transaction and observe that (1) the transaction summary token logo/amount is vertically aligned and (2) the "Sent/Received n" is aligned:

![image](https://user-images.githubusercontent.com/20442784/192846528-29f4e705-5f1c-40fd-b92a-95479c3f0929.png)

## Screenshots

See above
